### PR TITLE
Create Artifical Intelligence and add NLPCloud api

### PR DESCRIPTION
**Pull request by gaurovgiri**

### CODE REVIEW

File: src/content/apis/artificial-intelligence/nlpcloud.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: NLPCloud
+description: API for pre-trained or custom models for NER, sentiment-analysis, classification, summarization, paraphrasing, grammar and spelling correction, keywords and keyphrases extraction, chatbot, product description and ad generation, intent classification, text generation, image generation, blog post generation, code generation, question answering, automatic speech recognition, machine translation, language detection, semantic search, semantic similarity, tokenization, POS tagging, embeddings, and dependency parsing.
```
Comment (TheLearneer) [inline src/content/apis/artificial-intelligence/nlpcloud.md:3]: Please shorten the description 🙏 
Best to have to under 120 characters.

File: src/content/apis/artificial-intelligence/nlpcloud.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+categories:
+  - artificial-intelligence
```
Comment (TheLearneer) [inline src/content/apis/artificial-intelligence/nlpcloud.md:5]:
```suggestion
  - development
```

**Review by TheLearneer — CHANGES_REQUESTED** [review]

File: src/content/apis/artificial-intelligence/nlpcloud.md (hunk 1 above)
Comment (gaurovgiri) [reply src/content/apis/artificial-intelligence/nlpcloud.md:3]: sure

File: src/content/apis/artificial-intelligence/nlpcloud.md (hunk 2 above)
Comment (gaurovgiri) [reply src/content/apis/artificial-intelligence/nlpcloud.md:5]: should I rename the category folder to development as well?

Comment (TheLearneer) [reply src/content/apis/artificial-intelligence/nlpcloud.md]: Oh yes, there already is a development folder, so please move the file accordingly 🙇

**Review by TheLearneer — APPROVED** [review]
Thank you for your contribution @gaurovgiri 🙏

**Pull request merged**